src/main/java/taxi/controller/IndexController.java

```diff
@@ -10,6 +10,7 @@ public class IndexController extends HttpServlet {
     @Override
     public void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
+        req.setAttribute("login", req.getSession().getAttribute("login"));
```

Comment:
I added this because I didn't have any idea how to make things like this
![Screenshot](https://user-images.githubusercontent.com/90065495/231435356-72b6af1c-ac78-4734-a06d-133a33fc6ea2.png)

Reply:
Okay it's a redundant one